### DISCUSSION TITLE
关于 2022 年适配情况

### DISCUSSION THREAD

**Discussion by xjtu-blacksmith** [xjtu-blacksmith/easymcm#13] [General]
已注意到今年比赛动向。

近期较忙碌，计划在抽空出来后尽快确认目前模板是否适配今年样式，请少安毋躁。

**Comment by xjtu-blacksmith** [comment]
已检查完毕，基本没有变化。只是注意到纸张的尺寸是 US Letter 而非 A4，或许几年前的模板一直如此；今年更新时将其调整过来。除此之外，其他内容均保持不变。

新版本 v5.2.2 已发布，请各位用户检查。相关讨论贴：#14

**Reply by rla-research** [reply]
您好，不好意思打扰了，我想请问一下2022年的summary sheet是跟2020年一样的吗？因为我在阅读easy.msm的过程中发现在summary地方您标注说这是2020年的版本，请根据每年的变动自行调整。由于我已经无法找到2020年的美赛官网给的latex模版，所以想向您进行询问。

**Reply by xjtu-blacksmith** [reply]
> 您好，不好意思打扰了，我想请问一下2022年的summary sheet是跟2020年一样的吗？因为我在阅读easy.msm的过程中发现在summary地方您标注说这是2020年的版本，请根据每年的变动自行调整。由于我已经无法找到2020年的美赛官网给的latex模版，所以想向您进行询问。

应该是注释没改。现在的样式和今年的 summary sheet 是符合的。

**Reply by rla-research** [reply]
谢谢您🙏
